### ISSUE TITLE
Element actions take very long time in Android 6.0 and 7.0 when background animation is present in native app

### ISSUE THREAD

**Issue by puryarra** [appium/appium-uiautomator2-server#222]
sendKeys (AppiumDriver.setValue()) takes more than 2 minutes in spite of timeout of 35 seconds mentioned for waitForElementByAccessibilityId. when background animation is present in native app for Android versions 6.0 and 7.0. This also results abnormal ending of Appium session sometimes.
Works fine from Android ver 7.1.1 onwards with animation. 
Also no issue when animation is not present in app for versions 6.0 and 7.0
Tried with setting disableWindowAnimation capability to true, but same result.

Environment:

Node: 8.3.0
Appium server: 1.9.1
UIAutomator2 server: 1.18.0

Emulator logs can be found here. Same results for real device too.
[Android_Emulator_7.0.log](https://github.com/appium/appium-uiautomator2-server/files/2575650/Android_Emulator_7.0.log)

Appium log:
[Appium_7.0.log](https://github.com/appium/appium-uiautomator2-server/files/2575657/Appium_7.0.log)

**Comment by KazuCocoa** [comment]
Do you mean `disableWindowAnimation` does not work for Android API level 6.0 and 7.0?
If so, it probably fixes in https://github.com/appium/appium-uiautomator2-driver/pull/249/files

**Comment by puryarra** [comment]
> Do you mean `disableWindowAnimation` does not work for Android API level 6.0 and 7.0?
> If so, it probably fixes in https://github.com/appium/appium-uiautomator2-driver/pull/249/files

@KazuCocoa  Yes, `disableWindowAnimation` has no impact on Android 6.0 and 7.0 for the animation in our app. Thanks for the fix. I will try this build and get back.

**Comment by KazuCocoa** [comment]
Okay. Thanks. The latest `appium@beta` should work fine. Could you try with the latest `appium@beta`?
The version of uiautomator2-driver should be `1.24.0`. @puryarra

**Comment by vky26** [comment]
I have same issue, I use the below config  , trying to inspect an element in android react native app. Its taking more time to get inspected manually in Appium UI and failing when ran via code. 

Env:
Appium : 1.10.0
appium-uiautomator2-driver version - 1.28.0
Phone: Samsung S7 / OS 8.0
node: v10.1.0


Any pointers on this? 
@puryarra    did you get any luck on this?

@mykola-mokhnach can you help me here?

**Comment by KazuCocoa** [comment]
You could not disable animation, right?
Could you attach your appium server log to trach operations?

**Comment by vky26** [comment]
Hey @KazuCocoa I have attached log below. I use serenity framework, I have added the below to desired capabilities
appium.disableWindowAnimation = True

Its native app, there is animation running in the background which makes none of the element to be visible / no actions can be performed here. Manually inspecting through Appium GUI takes more than usual time(at least 1 minute) to identify each element and while running the scripts, its failing as not able to inspect element.

[Appium Logs.pdf](https://github.com/appium/appium-uiautomator2-server/files/2782390/Appium.Logs.pdf)

Again, I use same config as I stated before

Env:
Appium : 1.10.0
appium-uiautomator2-driver version - 1.28.0
Phone: Samsung S7 / OS 8.0
node: v10.1.0
PC: Mac OS 10.13
